utils/config.sh

```diff
@@ -0,0 +1,5 @@
+# File with demo configuration parameters
+
+export CP_VERSION_MAJOR=5.4
+export CP_VERSION_FULL=5.4.0
+export GH_BRANCH=5.4.0-post
```

Comment:
we might want to also consider adding something like `CP_JAR_VERSION=5.4.0`, which on `-x` branches could be similar to `5.4.0-SNAPSHOT`, or similar idea to this.  Possibly, in this change or later one, derive CP_JAR_VERSION's value entirely from the other variables.  We need a value like this in `microservices-orders/services-start.sh` and maybe elsewhere

Reply:
Good idea.  In latest commit I added JAR_VERSION.